src/OrbitDB.js

```diff
@@ -9,11 +9,12 @@ const DocumentStore = require('orbit-db-docstore')
 const Pubsub = require('orbit-db-pubsub')
 const Cache = require('orbit-db-cache')
 const Keystore = require('orbit-db-keystore')
+const IdentityProvider = require('orbit-db-identity-provider')
```

Comment:
After some initial user tests and fresh experiences on this, I realized that we should expose the IdentityProvider to the users so that they can actually set up custom identities :) Ie. export it like we do in ipfs-log.

Reply:
Take that back. User can do `const IdentityProvider = require('orbit-db-identity-provider')` whenever they use `orbit-db` since orbitdb pulls in `orbit-db-identity-provider`.